dex/src/critbit.rs

```diff
@@ -122,6 +134,34 @@ impl LeafNode {
     pub fn client_order_id(&self) -> u64 {
         self.client_order_id
     }
+
+    #[inline]
+    pub fn tif_offset(&self) -> u16 {
+        self.tif_offset
+    }
+
+    pub fn is_order_expired(
+        &self,
+        epoch_start_ts: u64,
+        start_epoch_seq_num: u64,
+        current_ts: u64,
+        is_bid: bool,
+    ) -> bool {
+        // Unset tif
+        if self.tif_offset() == 0 {
+            return false;
+        }
+
+        if epoch_start_ts + (self.tif_offset() as u64) < current_ts {
```

Comment:
if we roll over into the next epoch will this send out a false positive? i feel like we have to check seqnum as well in this statement

Reply:
this won't be an issue as we will check seq num if this condition fails anyway